modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,179 @@
+import cv2 as cv
+import glob
+import argparse
+import os
+
+# Method used for initializing of trackers by creating it 
+# via cv.TrackerX_create()
+def init_tracker(tracker_name):
+    config = {"Boosting": (cv.TrackerBoosting_create(), 500),
+    "MIL": (cv.TrackerMIL_create(), 1000),
+    "KCF": (cv.TrackerKCF_create(), 110),
+    "MedianFlow": (cv.TrackerMedianFlow_create(), 20),
+    "GOTURN": (cv.TrackerGOTURN_create(), 20),
+    "MOSSE": (cv.TrackerMOSSE_create(), 20),
+    "CSRT": (cv.TrackerCSRT_create(), 250)}
+    return config[tracker_name]
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Run LaSOT-based benchmark for visual object trackers")
+    # As a default argument used name of 
+    # original dataset folder
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="LaSOTTesting", help="Full path to LaSOT")
+    args = parser.parse_args()
+
+    # Creating list with names of videos via reading names from txt file
+    list_of_videos = [None for _ in range(280)]
+    video_names = os.path.join(args.path_to_dataset, "/testing_set.txt")
+    with open(video_names, 'rt') as f:
+        list_of_videos = f.read().rstrip('\n').split('\n')
+
+    trackers = [
+        'Boosting', 'MIL', 'KCF', 'MedianFlow', 'GOTURN', 'MOSSE', 'CSRT']
+
+    # Loop for every tracker
+    for tracker_id in range(len(trackers)):
+
+        tracker_name = trackers[tracker_id]
+        print(tracker_name)
+
+        # Loop for every video
+        for video_name in list_of_videos:
+
+            tracker, frames_before_reinit = init_tracker(tracker_name)
+            init_once = False
+
+            print("Video: " + str(video_name))
+
+            # Open specific video and read ground truth for it
+            video_name = video_name.replace("\n", "")
+            gt_file = open(
+                args.path_to_dataset + video_name + "/groundtruth.txt", "r")
+            gt_bb = gt_file.readline().replace("\n", "").split(",")
+            init_bb = gt_bb
+            init_bb = tuple([float(b) for b in init_bb])
+
+            print("Initial bounding box: ", init_bb)
+
+            # Creating blob from image sequence
+            video_sequence = sorted(glob.glob(str(args.path_to_dataset + str(
+                video_name) + "/img/*.jpg")))
+
+            print("Number of frames in video: " + str(len(video_sequence)))
+
+            # Variables for saving sum of every metric for every frame and
+            # every video respectively
+            sum_iou = 0
+            sum_pr = 0
+            sum_norm_pr = 0
+            iou_values = 0
+            pr_values = 0
+            norm_pr_values = 0
+            frame_counter = len(video_sequence)
+
+            # For every frame in video
+            for number_of_the_frame, image in enumerate(video_sequence):
+
+                for i in range(len(gt_bb)):
+                    gt_bb[i] = float(gt_bb[i])
+                gt_bb = tuple(gt_bb)
```

Comment:
Why we need tuple?
`init_gt = tuple([float(gt) for b in gt_bb])`

Reply:
Tracker require tuple as input.

Reply:
Trackers requires tuple for input.

Reply:
Please replace a loop into a single line:
```python
init_gt = tuple([float(gt) for b in gt_bb])
```

Reply:
OK.